docs/apps/features/02-airstreams.mdx

```diff
@@ -6,7 +6,7 @@ title: "Airstreams"
 
 # Sablier Airstreams
 
-The Sablier V2 interface can hadnle more than native streams. We developed a solution for distributing token streams to
+The Sablier V2 interface can handle more than native streams. We developed a solution for distributing token streams to
```

Comment:
FYI I suggest saying just `Sablier Interface`. We are moving away from the V2-tethering

cc @sablier-labs/frontend and @smol-ninja

Reply:
Sounds good to me.

Reply:
Since that would be in the future, shouldn't we stick to V2 terminology until we implement package tethering?